pkg/hintrunner/zero/zerohint_uint256.go

```diff
@@ -223,3 +223,99 @@ func createUint256SignedNNHinter(resolver hintReferenceResolver) (hinter.Hinter,
 	}
 	return newUint256SignedNNHint(a), nil
 }
+
+func newUint256UnsignedDivRemHint(a, div, quotient, remainder hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "Uint256UnsignedDivRem",
+		Op: func(vm *VM.VirtualMachine, _ *hinter.HintRunnerContext) error {
+			//> a = (ids.a.high << 128) + ids.a.low
+			//div = (ids.div.high << 128) + ids.div.low
+			//quotient, remainder = divmod(a, div)
+			//ids.quotient.low = quotient & ((1 << 128) - 1)
+			//ids.quotient.high = quotient >> 128
+			//ids.remainder.low = remainder & ((1 << 128) - 1)
+			//ids.remainder.high = remainder >> 128
+			aLow, aHigh, err := GetUint256AsFelts(vm, a)
+			if err != nil {
+				return err
+			}
+			var aLowBig big.Int
+			aLow.BigInt(&aLowBig)
+			var aHighBig big.Int
+			aHigh.BigInt(&aHighBig)
+
+			divLow, divHigh, err := GetUint256AsFelts(vm, div)
+			if err != nil {
+				return err
+			}
+			var divLowBig big.Int
+			divLow.BigInt(&divLowBig)
+			var divHighBig big.Int
+			divHigh.BigInt(&divHighBig)
+
+			a := new(big.Int).Add(new(big.Int).Lsh(&aHighBig, 128), &aLowBig)
+			div := new(big.Int).Add(new(big.Int).Lsh(&divHighBig, 128), &divLowBig)
+			quot, rem := utils.FeltDivRem(new(fp.Element).SetBigInt(a), new(fp.Element).SetBigInt(div))
```

Comment:
This should not be working properly, a uint256 number (256 bits) doesn't have to fit in a felt (252 bits). I think they do some compression when the number doesn't fit in a felt, but not sure how that works or how can it affect the outcome. See if you can use instead uint256 library as in #307, or if you can perform the operation as big.Int

Reply:
Yes, those are good. Another way was to use `DivMod()` function from bigInt library, but it probably does the exact same thing